modules/osm/node.js

```diff
@@ -130,14 +149,15 @@ Object.assign(osmNode.prototype, {
 
             Object.keys(nodeIds).forEach(function(nodeId) {
                 // +90 because geoAngle returns angle from X axis, not Y (north)
-                results.push(
-                    (geoAngle(this, resolver.entity(nodeId), projection) * (180 / Math.PI)) + 90
-                );
+                results.push({
+                    type: isSide ? 'side' : 'direction',
+                    angle: (geoAngle(this, resolver.entity(nodeId), projection) * (180 / Math.PI)) + (isSide ? 0 : 90)
```

Comment:
Good idea, though I wonder if reusing the “cone” would risk confusing some users. Some mappers are also using `side=*` on `traffic_signals=*` vertices. This change will cause these vertices’ cones to point away from the street, which would be inconsistent with the cones for standalone `traffic_signals=*` nodes. Plus, it might be necessary to combine `side=*` with `direction=*` in some cases.

Reply:
I was wondering about how to visualize the `side` as well. 

The `direction` is more of a "looking direction" or "affected area". 

But the `side` is more of "an anchor" or a "connection point" or a "this belongs to over there" indicator. However I don't see a way to visualize those well. Especially when the actual "anchor point" is unknown (we only now "it is to the right").

The other UI we use to show direction of elements is on `barrier=retaining_wall` and such with the small arrows next to the line. Maybe this is something we can build upon for this UI? Like one of those arrows next to the node?

Reply:
> Like one of those arrows next to the node?

Something like … <img width="40" alt="image" src="https://github.com/openstreetmap/iD/assets/111561/c7742ec3-c0d2-4e13-a37c-34c77df60d8d">?

Reply:
> Good idea, though I wonder if reusing the “cone” would risk confusing some users.
> [...]
> Maybe `side=*` could act as more of a mask, chopping the directional cone in half?

That's a cool idea, although it might be confusing if `side`+`direction` renders differently to `side` and `direction` on their own? Here's what it could look like:

`⊿`: ![](https://github.com/openstreetmap/iD/assets/16009897/3fb4c7c9-7d0e-47aa-a65e-219e8f92db51)
`◭`: ![](https://github.com/openstreetmap/iD/assets/16009897/bd29638a-2fda-47e6-9c77-db194b2d5ee7)

> Like one of those arrows next to the node?

Something like this maybe?

`△`: ![](https://github.com/openstreetmap/iD/assets/16009897/9b3f0d64-62fd-4ca6-87c4-5387f34d8083)
`→`: ![](https://github.com/openstreetmap/iD/assets/16009897/0d002f2d-346d-4b08-9106-32332572233c)
`⏢`: ![](https://github.com/openstreetmap/iD/assets/16009897/07a1b0a7-c212-4717-964e-c8e39e87dae2)
`▭`: ![](https://github.com/openstreetmap/iD/assets/16009897/eca6f5be-fe03-4b9a-b425-d41ae89519b0)
`T`: ![](https://github.com/openstreetmap/iD/assets/16009897/d1eee0bc-aa47-4180-be0a-b8c7b66c3b48)

Any other ideas would be greatly appreciated...